course7/homeworks/fred/complex-rpn/calculator.cc

```diff
@@ -0,0 +1,64 @@
+#include "calculator.h"
+
+#include <cassert>
+#include <stdexcept>
+
+namespace {
+
+  Complex calculate(Complex operand1, Operator op, Complex operand2) {
+    switch (op) {
+      case Operator::SUM:
+        return operand1 + operand2;
+
+      case Operator::SUBTRACTION:
+        return operand1 - operand2;
+
+      case Operator::MULTIPLICATION:
+        return operand1 * operand2;
+
+      case Operator::DIVISION:
+        return operand1 / operand2;
+
+      default:
```

Comment:
If you don't use the 'default' label, then the compiler will warn you if you don't use all the values of the enum.